hotosm/tasking-manager
# Adds ml-enabler integration

This adds ml-enabler integration to TM.

To test:
- Update requirements

- run looking-glass locally or update environment variables to point to a looking glass instance (see config.py)

- Create a project on step 3 you'll see the "ML Layer" button, click it to generate a prediction, this will take a while;

![Screen Shot 2019-07-24 at 11 06 38 PM](https://user-images.githubusercontent.com/9713/61847540-45a84c00-ae68-11e9-82b8-62e6e88f7009.png)

After the prediction finishes it should look like this:

![Screen Shot 2019-07-24 at 11 09 19 PM](https://user-images.githubusercontent.com/9713/61847557-4f31b400-ae68-11e9-9602-f999ad655e96.png)

After that the project can be created and task annotations will be created too so project detail will have the "ML Layer" button on too.
